neo/Consensus/ConsensusContext.cs

```diff
@@ -274,6 +273,7 @@ internal void EnsureMaxBlockSize(IEnumerable<Transaction> txs)
 
         public ConsensusPayload MakePrepareRequest()
         {
+            var random = new Random();
```

Comment:
I don't think that `MakePrepareRequest()` will be accessed by multiple threads at the same time. There should be no risk.

Reply:
I don't think so too, but always is better to use local, maybe in the future other developer use it and we forget this, i prefer to keep it local in order to be safe.

Reply:
We have alternatives. 

1- Local variable as now
2- Static variable (maybe in Helper.cs) using `ThreadStaticAttribute`
```csharp
        [ThreadStatic]
        private static Random _rand;

        /// <summary>
        /// Thread safe Random
        /// </summary>
        private static Random Rand
        {
            get
            {
                if (_rand == null) _rand = new Random();
                return _rand;
            }
        }
```

Reply:
`ThreadStaticAttribute` will create different values for each thread. It is not a good option.

We only need one `Random`. Although there may be multiple threads accessing it, different threads are not accessed at the same time, so it is safe.

Reply:
> so it is safe.

It's safe now, but dangerous, I preffer to use a local variable, the speed will be almost the same.